src/content/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/create-nrql-alert-conditions.mdx

```diff
@@ -16,62 +16,157 @@ redirects:
   - /docs/alerts/new-relic-alerts/defining-conditions/create-alert-conditions-nrql-queries
 ---
 
-import queriesnrqlAlertConditions from 'images/queries-nrql_screenshot-full_alert-conditions.png'
+import queriesPolicyNewAlertCondition from 'images/queries-nrql_screenshot-crop_create-alert-condition.png'
 
-import queriesnrqlNrqlContextDemo from 'images/queries-nrql_screenshot-full_nrql-context-demo.gif'
+import queriesMetricExplorer from 'images/queries-nrql_screenshot-crop_metrics-create-alert-condition.png'
 
 import queriesnrqlSignalLossUi from 'images/queries-nrql_screenshot-full_signal-loss-ui.png'
 
 import queriesnrqlFineTuneSignals from 'images/queries-nrql_screenshot-full_fine-tune-signals.png'
 
-You can use [NRQL queries](/docs/insights/new-relic-insights/using-new-relic-query-language/using-nrql) to create alert conditions. Once you've defined your signal, you can further define your warning and critical threshold levels. This determines when an alerts incident is created.
+You can use [NRQL queries](/docs/insights/new-relic-insights/using-new-relic-query-language/using-nrql) to create alert conditions. Once you've defined your signal, you can further define your warning and critical threshold levels. This determines when an alerts incident is created. For more information on key concepts relating to NRQL alert conditions and streaming alerts, see [Streaming alerts: key terms and concepts](/docs/alerts-applied-intelligence/new-relic-alerts/get-started/streaming-alerts-key-terms-concepts).
```

Comment:
One thing I've been confused about with NRQL alert conditions: what is the customer-facing story around these? For example, I once heard that all of our alert conditions would one day be NRQL conditions. Is that true? How often will customers have to make these? It might be good to explain how often they'll be doing it, and when they'll be doing this in comparison to other alert types (which maybe they're all NRQL these days, I don't really know what's going on with them).

Reply:
You're right. I will ask for more info about it and if it's essential to add it, I'll create a new PR. Thanks!